### ISSUE TITLE
BiocycToCSV: ontology_text and features_at_acetylation_site columns.

### ISSUE THREAD

**Issue by TheYavimayan** [coffeetocode/bioinfoscripts#4]
There is just one thing that I think would really put the cherry on top. Is it possible to put each "ontology text" and "features at acetylation site" as a separate column? You might have noticed that it gets hard to read some of the longer strings of values.

For example: SelD (EG10943) has the following GO terms in the single cell, making it hard to read:
cytosol;transferase activity;kinase activity;protein homodimerization activity;magnesium ion binding;selenocysteine biosynthetic process;tRNA seleno-modification;phosphorylation;ATP binding;nucleotide binding;selenide, water dikinase activity;cytoplasm

Similarly, FabI (EG11528) has a "features_at_acetylation_site" column that gets a bit messy:
13-252/no comment;205-205/UniProt: Causes a further reduction in kcat/Km for reduction of DD-ACP without affecting kcat/Km for the DD-CoA substrate. Has a larger impact on substrate reduction.;205-205/UniProt: No effect on substrate reduction.;2-262/UniProt: Enoyl-[acyl-carrier-protein] reductase [NADH] FabI.

I don't know what that might entail, but I'm hoping it's straightforward.

Thanks again!

**Comment by coffeetocode** [comment]
Both of those columns are the result of the features and ontology terms in the incoming XML data having a one-to-many relationship to the GENEID (that's actually probably a main reason they don't expose the data as CSV; it wouldn't capture the complexity of the relationships in the underlying data).

So the question is, specifically what should the result look like? If EG10943 has 6 ontology terms, and EG10020 has 27... then how many ontology term columns should there be? So, the simplest possible answer (and the one that I took here as a temporary measure) was: There should be only one column, with all of the different terms crammed together with semicolons. 

Another easy and slightly more readable approach could be to usenewlines instead of semicolons, so something like:
![image](https://cloud.githubusercontent.com/assets/1274965/17311215/20c91b24-580f-11e6-8327-af143490cc47.png)

Basically the same thing, all crammed in one cell, just easier for humans here. 

Alternately, if you want a variable number of columns depending on the maximum number of GO terms found for a gene, that's a way we could go to. So "ontology_text_0", "ontology_text_1", etc up to the max number we found. Totally doable, though I don't think the results are actually any more manageable, and you'll have a ton of columns that are unused by the genes that have only a few ontology terms.

Welcome to database design 101; just trying to figure out what to from data is enough to start pulling your hair out.
